pkg/agent/datamodel/addons.go

```diff
@@ -0,0 +1,1066 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
addons are for control plane, we don't need it in agentbaker.

Reply:
Hi, Tony,
Thanks for your review. I'll send out a follow PR to remove code not used after this goes in.

For addons.go, there is a function "setAddonsConfig" used in defaults.go's "func (cs *ContainerService) setOrchestratorDefaults(isUpgrade, isScale bool)":
		// Configure addons
		cs.setAddonsConfig(isUpgrade)

are you saying we can remove that line from defaults.go as well?

Reply:
I think so.

Reply:
PR to remove setAddonsConfig and other not used functions is here:
https://github.com/Azure/AgentBaker/pull/192